frontend/src/pages/admin.js

```diff
@@ -260,6 +266,18 @@ class Admin extends React.Component {
                                 )
                               }
                             />
+                            <IconButton
+                              icon={faUpload}
+                              size="sm"
+                              title={"Upload data"}
+                              onClick={(e) =>
+                                this.handleUploadData(
+                                  e,
+                                  "admin",
```

Comment:
@manrajgrover Need some clarifications here. Is the assumption that the `admin` is always uploader of data correct?

Reply:
Yes, users shouldn't be allowed to upload data and this feature should only be available to admin through admin portal.